chainbridge/src/tests.rs

```diff
@@ -273,7 +273,7 @@ fn create_sucessful_proposal() {
             votes_for: vec![RELAYER_A],
             votes_against: vec![],
             status: ProposalStatus::Initiated,
-            expiry: ProposalLifetime::get(),
+            expiry: ProposalLifetime::get()+1,
```

Comment:
This is changed since the new test externalities are set with block 1. With Block 0, events are not captured and test panicked.